config/application.rb

```diff
@@ -47,6 +47,9 @@ class Application < Rails::Application
     # TODO Disable this. Uncommenting this line will currently cause rspec to fail.
     config.action_controller.permit_all_parameters = true
 
+    config.active_job.queue_adapter     = :resque
+    config.active_job.queue_name_prefix = "foodsoft_#{Rails.env}"
```

Comment:
Can we include an environment variable, so that one could run more easily multiple instances of Foodsoft with one Redis instance? e.g.

```ruby
config.active_job.queue_name_prefix = ENV.fetch('JOB_QUEUE_PREFIX', "foodsoft_#{Rails.env}")}
```